src/daft-dsl/src/lit.rs

```diff
@@ -57,6 +57,8 @@ pub enum LiteralValue {
     Date(i32),
     /// A 64-bit floating point number.
     Float64(f64),
+    /// A list
+    List(Vec<LiteralValue>),
```

Comment:
Instead of having a `Vec<LiteralValue>`, We should instead just have a SeriesLiteral. This will allow you to avoid perform all the parsing and validation yourself. In the python side you should just be able to do `Series.from_*` to support a variety of input types such as a list, series, arrow_array, etc

Reply:
awesome! just did this in latest commit